src/sdk/conference/channel.js

```diff
@@ -219,8 +219,16 @@ export class ConferencePeerConnectionChannel extends EventDispatcher {
             transceiverInit.sendEncodings = options.video;
             this._videoCodecs = videoCodecs;
           }
-          this._pc.addTransceiver(stream.mediaStream.getVideoTracks()[0],
+          const transceiver = this._pc.addTransceiver(stream.mediaStream.getVideoTracks()[0],
             transceiverInit);
+
+          if (Utils.isFirefox()) {
+            // Firefox does not support encodings setting in addTransceiver
```

Comment:
Ends with period.

Also add the link for Firefox bug https://bugzilla.mozilla.org/show_bug.cgi?id=1396918.